compositor_pipeline/src/error.rs

```diff
@@ -20,6 +20,9 @@ pub enum RegisterOutputError {
 
     #[error("Encoder error while registering output stream for stream \"{0}\".")]
     EncoderError(OutputId, #[source] OutputInitError),
+
+    #[error("Failed to register input stream. Stream \"{0}\" has an odd resolution.")]
```

Comment:
What about the output stream?
Don't use odd here, it sounds weird, maybe "resolution is not divisible by 2" or
```
Failed to register input stream \"{0}\". Resolution in each dimension has to be divisible by 2."
```

Reply:
This should be `output_stream`. We can't return error in case of input stream, since resolution of input stream is dynamic